cluster/manager.go

```diff
@@ -141,14 +141,22 @@ func (c *MemberlistManager) setList(list *memberlist.Memberlist) {
 }
 
 func (c *MemberlistManager) ThisNode() Node {
+	return c.thisNode()
+}
+
+func (c *MemberlistManager) thisNode() HTTPNode {
```

Comment:
why do we need this private function if we already had the public one that did the same?

Reply:
right, that's not necessary anymore, was only necessary in some previous version :+1:

Reply:
https://github.com/grafana/metrictank/pull/555/commits/521206c90abb468401cbae77e0fd767e01f76b59